### PR TITLE
WE-166 Single Page Template

### PR DESCRIPTION
**Pull request by logansparlin**
## What Changed

- Added new DocumentationPage template
- Upgrade Matchbox to 6.1.0

## How to verify
- Run locally or visit deploy link and visit any momentum page.
- To verify the last updated styled go to `pages/momentum/[slug].tsx` and pass in a random string like `September 2021` to the `lastUpdated` prop

### CODE REVIEW

File: components/site/breadcrumbs.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { useRouter } from 'next/router';
+import { Breadcrumb } from '@sparkpost/matchbox';
+
+type BreadcrumbProps = {
+  title?: string;
+}
+
+const urlMap: {[key: string]: string} = {
+  'momentum': 'Momentum Documentation',
```
Comment (jonambas) [inline components/site/breadcrumbs.tsx:9]: Oh this is cool, should we use this in the Algolia dropdown too?

Comment (logansparlin) [reply components/site/breadcrumbs.tsx:9]: Yea we should, that would be nice

Comment (EvanYoung-SP) [reply components/site/breadcrumbs.tsx:9]: I guess my only comment here then would be if we do use this for algolia too, we should make this more of a global value so we're updating it once in one place. :-)

**Review by EvanYoung-SP — APPROVED** [review]
The comment I have doesn't have to be accomplished in this PR, so I'm going to mark it as approved.

**Pull request merged**